errors/error.go

```diff
@@ -47,6 +47,11 @@ For operations that change state/configuration of the cluster (e.g. "cluster cha
 To change CLUSTER_WRITE permission, see the documentation: https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/rest-api#using-rest-endpoint-groups`
 )
 
+var (
+	ErrUserCancelled = errors.New("user cancelled")
+	ErrNotDecoded    = errors.New("not decoded")
```

Comment:
I couldn't find any occurrences for these two error definitions. Will they be used later on in refactoring stage two or three or should we remove them?

Reply:
RIght, they will be used in the serialization related PR.